.github/workflows/deploy_main.yml

```diff
@@ -29,3 +29,5 @@ jobs:
         env:
           RELEASE_BRANCHES: master
```

Comment:
The current release branch is the "main", change the statement to RELEASE_BRANCHES: master,main